src/main/asciidoc/reference/mapping.adoc

```diff
@@ -39,6 +44,15 @@ By creating your own instance, you can register Spring converters to map specifi
 
 You can configure the `MappingR2dbcConverter` as well as `DatabaseClient` and `ConnectionFactory` by using  Java-based metadata. The following example uses Spring's Java-based configuration:
 
+If you set `setForceQuote` of the `R2dbcMappingContext to `true` table and column names derived from classes and properties will be used with database specific quotes.
+This means it is ok to use reserved SQL words like for example `order` in these names.
```

Comment:
Change to:

This means that it is OK to use reserved SQL words (such as `order`) in these names.